samples/zigbee/light_bulb/boards/nrf5340dk_nrf5340_cpuapp.conf

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Use software cryptography on nRF5340
+CONFIG_ZIGBEE_USE_SOFTWARE_AES=y
+CONFIG_CRYPTO=y
+CONFIG_CRYPTO_NRF_ECB=y
```

```suggestion
CONFIG_CRYPTO=n
CONFIG_CRYPTO_NRF_ECB=n
```
`prj.conf` will still include `CONFIG_CRYPTO_INIT_PRIORITY=80` so need tho check if warning will fail the test plan

---

We checked with Maciej recently about this and twister does not support failing on Kconfig warnings, unless there is another script being used for that?

---

I've checked Zigbee nightly plan and twister warnings don't affect the plan.

---

Also, requested changes in this file are still valid. For nRF5340 we don't need there CYPTO KConfig options, since `prj.conf` enables them, it makes sense to disable them in overlay for nRF5340 (as this is disabled in the `network_ccordinator` sample)

---

Good catch, seems I forgot to invert the options, fixed.